sklearn/linear_model/_bayes.py

```diff
@@ -118,6 +118,18 @@ class BayesianRidge(RegressorMixin, LinearModel):
     n_iter_ : int
         The actual number of iterations to reach the stopping criterion.
 
+    X_offset_ : array-like (length = n_samples)
```

Comment:
you can see how to write these lines on https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-documentation